Formula/w/whisper-cpp.rb

```diff
@@ -0,0 +1,30 @@
+class WhisperCpp < Formula
+  desc "Port of OpenAI's Whisper model in C/C++"
+  homepage "https://github.com/ggerganov/whisper.cpp"
+  url "https://github.com/ggerganov/whisper.cpp/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "b2e34e65777033584fa6769a366cdb0228bc5c7da81e58a5e8dc0ce94d0fb54e"
+  license "MIT"
+  head "https://github.com/ggerganov/whisper.cpp.git", branch: "master"
+
+  resource("samples_jfk.wav") do
+    url "https://github.com/ggerganov/whisper.cpp/raw/v1.4.0/samples/jfk.wav"
+    sha256 "59dfb9a4acb36fe2a2affc14bacbee2920ff435cb13cc314a08c13f66ba7860e"
+  end
+  resource("for-tests-ggml-tiny.bin") do
+    url "https://github.com/ggerganov/whisper.cpp/raw/v1.4.0/models/for-tests-ggml-tiny.bin"
+    sha256 "c486fb9f14a28b1c1dc252741a431646cc573450c900b9d9c406e10294aa01e6"
+  end
```

Comment:
Do these need to stay in sync with the release?

Reply:
Not necessarily. Unless the model format changes, I think this would remain as is.

Reply:
Can you prefix the resource names with `homebrew-` then? That will prevent our CI from requesting updates for it on every release.

Reply:
We can also replace the sample_jfk.wav resource with `testpath.install test_fixtures("test.wav")` . I tried to do so and tested it locally, but for some reason it wasn't found in `/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/test/support/fixtures/`, despite being present in [the repo](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/support/fixtures/test.wav)... How can we solve that?